bbr.py

```diff
@@ -165,7 +166,9 @@ def main():
     parser.add_argument("--bw", choices=[10, 20, 50, 100, 250, 500, 750, 1000], default=10,
                         help="Bandwidth for the bottleneck link", type=int, dest="bw")
     parser.add_argument("-s", "--size", "--buffer-size", choices=[0.1, 1, 10, 20, 50], default=0.1,
-                        help="Switch buffer size", type=float, dest="size")
+                        help="Switch buffer size", type=float,
+                        dest="buffer_size")
+    # TODO(liwenbo): should the default be "" instead?
     parser.add_argument("--remote-host", default="localhost", type=str, dest="remote_host",
```

Comment:
The remote code will behave differently based on whether it's `localhost` or not.